packages/twenty-front/src/pages/settings/profile/appearance/components/SettingsAppearance.tsx

```diff
@@ -1,14 +1,24 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
Could we please rename SettingsAppearance to SettingsExperience to reflect the current title?